yare-model-converters/src/main/java/com/sabre/oss/yare/common/converter/aliases/TypeAliases.java

```diff
@@ -31,23 +31,32 @@
 import java.util.Properties;
 import java.util.stream.Collectors;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 final class TypeAliases {
-    private static final String ALIAS_CONFIGURATION_RESOURCE_FILE =
-            "com/sabre/oss/yare/common/converter/aliases/typeAliases.properties";
+    private static final Logger log = LoggerFactory.getLogger(TypeAliases.class);
+
+    static final String ALIAS_CONFIGURATION_RESOURCE_FILE = "typeAliases.properties";
```

Comment:
There has been removed "com/sabre/oss/..." from this variable. It can introduce conflicts with client's code (e.g. different libraries used be client).